cmd/uhc/cluster/list/list.go

```diff
@@ -35,33 +37,39 @@ var args struct {
 	debug     bool
 	managed   bool
 	step      bool
+	columns   string
+	padding   int
 }
 
 var managed bool
 
+// NONE Constant for empty value in columns:
+var NONE = "NONE"
```

Comment:
This can be a constant, and can be private, so I suggest:

```
const none = "NONE"
```

Reply:
no need